non-oss/pharos_pro/addons/kontena-lens/resources/07-dashboard-deployment.yml.erb

```diff
@@ -14,6 +14,21 @@ spec:
       labels:
         app: dashboard
     spec:
+      <%- unless node_selector.empty? -%>
```

Comment:
Why not just `if config.node_selector` and then `config.node_selector.to_h.each`?

Reply:
Yes, good question. Just copied this part from nginx-ingress-controller addon. I will change this and fix this also for nginx controller in separate PR